demos/python_demos/whiteboard_inpainting_demo/models.lst

```diff
@@ -0,0 +1,3 @@
+# This file can be used with the --list option of the model downloader.
```

Comment:
Why do you need the `__init__.py` file in this directory? Can't leave a comment for this change, 'cause it's empty, so leaving it here.